packages/run-protocol/src/collectFees.js

```diff
@@ -0,0 +1,31 @@
+// @ts-check
+/**
+ * Provide shared support for providing access to fees from a service contract.
+ *
+ * @param {ZCF} zcf
+ * @param {ZCFSeat} feeSeat
+ * @param {Brand} feeBrand
+ * @param {string} keyword
+ * @returns {{makeCollectFeesInvitation: (function(): *)}}
+ */
+export const makeMakeCollectFeesInvitation = (
```

Comment:
`makeMake` 🙄

Reply:
Not my name. I just moved it. :)

Reply:
sure, but if there's a better name this is a good time to change it.

It's common to have chained makers. is prepending "make" the pattern to follow? (I thought there must be something better but I can't think of one. I've asked in chat.)